### PR TITLE
get rid of wide fields in wide combiner. Not used in llvm

### PR DESCRIPTION
**Pull request by lll-phill-lll**
### Changelog entry 

...

### Changelog category 

* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp [hunk 1]
```diff
@@ -346,16 +364,16 @@ class TSpillingSupportState : public TComputationValue<TSpillingSupportState> {
     enum class ETasteResult: i8 {
         Init = -1,
         Update,
-        Skip
+        DataRequired,
```
Comment (Tony-Romanov) [inline ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:367]: Я бы назвал их ConsumeRawData  и ExtractRawData

Comment (lll-phill-lll) [reply ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp]: да, поддерживаю, спасибо

**Review by Tony-Romanov — APPROVED** [review]

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp [hunk 2]
```diff
@@ -1237,6 +1250,7 @@ using TBaseComputation = TStatefulWideFlowCodegeneratorNode<TWideLastCombinerWra
         , AllowSpilling(allowSpilling)
     {}
 
+    // MARK: DoCAlculate
```
Comment (Darych) [inline ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:1253]: Do we need this mark?

**Review by Darych — APPROVED** [review]

File: ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp (hunk 2 above)
Comment (lll-phill-lll) [reply ydb/library/yql/minikql/comp_nodes/mkql_wide_combine.cpp:1253]: No, forgot to remove it 😢

**Pull request merged**